src/utils/api/handlerUtils.js

```diff
@@ -0,0 +1,13 @@
+const methodGET = R.compose(R.equals('GET'), R.propOr(null, "method"))
+const methodPOST = R.compose(R.equals('POST'), R.propOr(null, "method"))
+const methodPUT = R.compose(R.equals('PUT'), R.propOr(null, "method"))
+const methodDELETE = R.compose(R.equals('DELETE'), R.propOr(null, "method"))
+
+const resourceBase = R.compose(R.equals('/'), R.propOr(null, "resource"))
+export const createResourceTest = (resource) => R.compose(R.equals(resource), R.propOr(null, "resource"))
+
+// expect an object with resource and method keys e.g. { resource: '/', method: 'GET'g}
+export const getAPI = R.allPass([methodGET, resourceBase]);
```

Comment:
It doesn't, it was just as an example ...you might have 
`
const resourceInvoice = R.propEq('resource', 'invoice');`


and then 
`export const getInvoice = R.allPass([methodGET, resourceInvoice]);`